GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1084,49 +1074,31 @@ - (void)OAuthLoginWithAddScopesFlow:(BOOL)addScopesFlow
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
               completion:completion];
     } else {
+      // Mock `maybeFetchToken:` method in Sign in flow.
+      if (!(authError || modalCancel)) {
+        [[[_authState expect] andReturn:nil] lastTokenResponse];
+    #if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
```

Comment:
Remove indentation for preprocessor directives.

Reply:
Done.